packages/cli/src/projects.ts

```diff
@@ -50,3 +73,31 @@ async function switchProject(medplum: MedplumClient, projectId: string): Promise
     console.log(`Switched to project ${projectId}\n`);
   }
 }
+
+async function inviteUser(
+  projectId: string,
+  firstName: string,
+  lastName: string,
+  email: string,
+  resourceType: string,
+  sendEmail: boolean,
+  admin: boolean
+): Promise<void> {
+  const body = {
+    firstName,
+    lastName,
+    resourceType,
+    email,
+    sendEmail,
+    admin,
+  };
+  try {
+    await medplum.post('admin/projects/' + projectId + '/invite', body);
```

Comment:
You should be able to now use `medplum.invite()`, which would provide some type safety.

It might make sense to align this function signature with that signature as well.  That's more of a style choice though, your call.